packages/react-docs/example/ts/docs.tsx

```diff
@@ -0,0 +1,123 @@
+            DecodedLogEntryEvent: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L150',
+            LogEntryEvent: 'http://mikemcl.github.io/bignumber.js',
+            CallData: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L348',
+            BlockWithoutTransactionData:
+                'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L314',
+            LogEntry: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L366',
+            FilterObject: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L109',
+            ['Web3.BlockParam']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L278',
+            ['Web3.ContractAbi']: 'https://github.com/0xProject/web3-typescript-typings/blob/f5bcb96/index.d.ts#L47',
+        },
+        typeNameToPrefix: {
+            Provider: 'Web3',
+            CallData: 'Web3',
+            BlockWithoutTransactionData: 'Web3',
+            LogEntry: 'Web3',
+            FilterObject: 'Web3',
+        },
+        typeNameToDocSection: {
+            Web3Wrapper: docSections.web3Wrapper,
+        },
+    },
+};
+const docsInfo = new DocsInfo(docsInfoConfig);
+
+const availableVersions = ['0.1.12', '0.2.0'];
+const versionToDocJSON: { [semver: string]: object } = {
+    [availableVersions[0]]: v0TypeDocJson,
+    [availableVersions[1]]: v2TypeDocJson,
+};
+
+export interface DocsProps {}
+
+interface DocsState {
+    selectedVersion: string;
+    docAgnosticFormat?: DocAgnosticFormat;
+}
+
+export class Docs extends React.Component<DocsProps, DocsState> {
+    constructor(props: DocsProps) {
+        super(props);
+        this.state = {
+            selectedVersion: availableVersions[1],
+            docAgnosticFormat: docsInfo.convertToDocAgnosticFormat(v2TypeDocJson),
+        };
+    }
+    public render() {
+        const menuSubsectionsBySection = _.isUndefined(this.state.docAgnosticFormat)
+            ? {}
+            : docsInfo.getMenuSubsectionsBySection(this.state.docAgnosticFormat);
+        return (
+            <Documentation
+                selectedVersion={this.state.selectedVersion}
+                availableVersions={availableVersions}
+                docsInfo={docsInfo}
+                docAgnosticFormat={this.state.docAgnosticFormat}
+                sourceUrl={this._getSourceUrl()}
+                onVersionSelected={this._onVersionSelected.bind(this)}
+            />
+        );
+    }
+    private _onVersionSelected(semver: string) {
+        const selectedDocJSON = versionToDocJSON[semver];
+        this.setState({
+            selectedVersion: semver,
+            docAgnosticFormat: docsInfo.convertToDocAgnosticFormat(selectedDocJSON as TypeDocNode),
+        });
+    }
+    private _getSourceUrl() {
+        const sourceUrl = `${docsInfoConfig.packageUrl}/blob/@0xproject/web3-wrapper%40${
```

Comment:
does this need to be url encoded in code or will it work without it and the browser will fix?

Reply:
Browser can fix.